### DISCUSSION TITLE
Welcome to GrupppenarbeitWA Discussions!

### DISCUSSION THREAD

**Discussion by yannick0404** [yannick0404/GrupppenarbeitWA#19] [Announcements]
Diskussion für den Abschluss des Projektes 🚀

**Comment by Stammkundschaft** [comment]
Ich fange mal an:

Alter schwankt im Datensatz so um die 25 Jahre rum, wobei es nicht wirklich große Ausreißer gibt (Range von 20 bis 30 Jahren).
Leute, die im Mathe-LK waren, sind im Schnitt aelter als die, die nicht drin waren.
Aus dem Korrelationsplot kann man einen leicht positiven Zusammenhang zwischen dem Alter und dem Studienfach Info herauslesen.

**Reply by yannick0404** [reply]
Ja, das Alter ist um den Erwartungswert 25 mit einer Standardabweichung von 2 verteilt.
Es ist allerdings kein Zusammenhang zwischen Alter und dem Studienfach Informatik gewollt simuliert worden. Meiner Meinung nach ist auch eine Korrelation von ca. 0.23 nicht wirklich Aussagekräftig, wie du bereits erwähntest.

**Reply by yannick0404** [reply]
Okey da die replies anscheinend untergehen machen wir da wahrscheinlich besser so, wie @edos1 das gemacht hat.
